### PR TITLE
Version lock for argon2 and start-full-service tweaks

### PR DESCRIPTION
**Pull request by sugargoat**
Soundtrack of this PR: [While Strolling through the Park One Day](https://youtu.be/noBLXQY4yuA)

### Motivation

Pinning argon2 version and fixing up start-full-service script for 

### In this PR

- Pins argon2 to `0.28.1`
- Makes logging less verbose for full-service

### CODE REVIEW

File: app/package.json [hunk 1]
```diff
@@ -17,6 +17,6 @@
     "@types/i18next-node-fs-backend": "^2.1.0"
   },
   "dependencies": {
-    "argon2": "^0.27.2"
+    "argon2": "^0.26.2"
```
Comment (voloshyn) [inline app/package.json:20]: is `0.28.1` a typo in the PR description?

File: full-service-bin/mac/start-full-service.sh [hunk 2]
```diff
@@ -23,15 +23,15 @@ mkdir -p "${WALLET_DB_DIR}"
...
-./full-service-testnet \
+RUST_LOG=debug,mc_connection=info,mc_ledger_sync=info ./full-service-testnet \
         --ledger-db "${LEDGER_DB_DIR}" \
         --wallet-db "${WALLET_DB_FILE_DIR}" \
         --poll-interval 1 \
         --peer mc://node1.test.mobilecoin.com/ \
         --peer mc://node2.test.mobilecoin.com/ \
         --tx-source-url https://s3-us-west-1.amazonaws.com/mobilecoin.chain/node1.test.mobilecoin.com/ \
         --tx-source-url https://s3-us-west-1.amazonaws.com/mobilecoin.chain/node2.test.mobilecoin.com/ \
-        --fog-ingest-enclave-css ./../ingest-enclave.testnet.css \
```
Comment (davidyhall) [inline full-service-bin/mac/start-full-service.sh:34]: I don't know if this is incorrect.

File: package.json [hunk 3]
```diff
@@ -213,6 +213,7 @@
     "@types/yup": "^0.29.7",
     "@typescript-eslint/eslint-plugin": "^4.12.0",
     "@typescript-eslint/parser": "^4.12.0",
+    "argon2": "^0.26.2",
```
Comment (davidyhall) [inline package.json:216]: This shouldn't be here.

**Review by davidyhall — CHANGES_REQUESTED** [review]

**Review by davidyhall — APPROVED** [review]
lgtm

**Pull request merged**